app/parse.py

```diff
@@ -8,5 +13,26 @@ class Course:
     duration: str
 
 
+def get_course(course: Tag) -> Course:
+    return Course(
+        name=course.select_one(".ProfessionCard_title__Zq5ZY > h3").text,
```

Comment:
Ensure that the CSS selector `.ProfessionCard_title__Zq5ZY > h3` is correct and matches the structure of the webpage. If the webpage structure changes, this selector might break.